elfo-configurer/src/protocol.rs

```diff
@@ -29,6 +29,6 @@ pub struct ReloadConfigsRejected {
 #[message(part)]
 #[non_exhaustive]
 pub struct ReloadConfigsError {
-    // TODO: meta
+    pub group: Option<String>,
```

Comment:
Should it be mandatory with "configurers" (actually the real name according to topology) if the config file is invalid?